src/SprykerEco/Shared/AmazonQuicksight/Transfer/amazon_quicksight.transfer.xml

```diff
@@ -279,4 +279,17 @@
         <property name="userName" type="string"/>
     </transfer>
 
+    <transfer name="QuicksightDeleteDataSetRequest" strict="true">
+        <property name="awsAccountId" type="string"/>
+        <property name="dataSetId" type="string"/>
+    </transfer>
+
+    <transfer name="QuicksightDeleteDataSetResponse" strict="true">
+        <property name="errors" type="Error[]" singular="error"/>
+    </transfer>
+
+    <transfer name="QuicksightDeleteAssetBundleDataSetsResponse" strict="true">
+        <property name="errors" type="Error[]" singular="error"/>
```

Comment:
Do we need to set success status explicitly? usually if there are no errors it's not automatically means that it succeeded

Reply:
makes sense but it is done for all the transfers in this way so I would leave as it to keep it consistent